scripts/gsp-local.sh

```diff
@@ -99,7 +99,7 @@ minikube start \
 	--memory ${GSP_MEMORY:-8192} \
 	--cpus ${GSP_CPUS:-4} \
 	--disk-size 30g \
-	--vm-driver hyperkit \
+  --vm-driver ${VM_DRIVER:-hyperkit} \
```

Comment:
Have you lost a space? Otherwise looks fine.

Reply:
Yeah, my editor and whoever else's editor who last edited this have disagreed. :grimacing: